pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -36,8 +37,54 @@ def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool,
         self.logger = runtime.logger
         self.working_dir = self.runtime.working_dir
         self.fail_count_name = f'{version}-assembly.{assembly}.count'
+        self.job_run = self.runtime.get_job_run_url()
+
+    async def comment_on_assembly_pr(self):
+        """
+        Comment the link to this jenkins build on the assembly PR if it was triggered automatically
+        """
+        owner = "openshift-eng"
+        repository = "ocp-build-data"
+        branch = self.doozer_data_gitref
+        token = os.environ.get('GITHUB_TOKEN')
+
+        api = GhApi(owner=owner, repo=repository, token=token)
+
+        # Head needs to have the repo name prepended for GhApi to fetch the correct one
+        head = f"{owner}:{branch}"
```

Comment:
I meant the head can be constructed with `<user_from_doozer_data_path>:<branch>`
In cases where gen-assembly triggers it 
the default data path would be `openshift-eng/ocp-build/data` 
and branch would be `auto-gen-assembly..`

and in other case when someone manually runs it, it would be `thegreyd/ocp-build-data`
and branch `assembly-xyz`
and so we would still be able to find the corresponding pull request and comment

Reply:
The name should be the github user coming from doozer data path job param

Reply:
ahh yes, sorry bit sleep deprived today, will make the change

Reply:
Success! https://github.com/openshift-eng/ocp-build-data/pull/3389